course/src/main/java/org/openedx/course/presentation/handouts/HandoutsViewModel.kt

```diff
@@ -3,23 +3,31 @@ package org.openedx.course.presentation.handouts
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.launch
 import org.openedx.core.BaseViewModel
+import org.openedx.core.SingleEventLiveData
+import org.openedx.core.config.Config
 import org.openedx.core.domain.model.AnnouncementModel
 import org.openedx.core.domain.model.HandoutsModel
 import org.openedx.course.domain.interactor.CourseInteractor
-import kotlinx.coroutines.launch
 
 class HandoutsViewModel(
     private val courseId: String,
+    private val config: Config,
     private val handoutsType: String,
     private val interactor: CourseInteractor
 ) : BaseViewModel() {
 
+    private val _apiHostUrl = MutableLiveData<String>()
+    val apiHostUrl: LiveData<String>
+        get() = _apiHostUrl
+
```

Comment:
If it remains constant and there's no requirement to observe changes, could it be a simple `val` variable?

Reply:
used getter instated of `MutableLiveData`.